README.md

```diff
@@ -1331,6 +1332,19 @@ Total: 0 (UNKNOWN: 0, LOW: 0, MEDIUM: 0, HIGH: 0, CRITICAL: 0)
 $ trivy --cache-dir /tmp/trivy/ image python:3.4-alpine3.9
 ```
 
+### Specify cache backend
+Trivy supports local filesystem and Redis as the cache backend. This option is useful especially for client/server mode.
+
+Two options:
```

Comment:
Actually, I also wanted to do that. But the problem is that the vulnerability database also depends on `--cache-dir` now.
1. vulnerability database
2. image/layer cache

`--cache-dir` changes the directory of the above two files. `--cache-backend` controls only `#2` since `#1` doesn't support Redis. I know it is a bit confusing, but I couldn't come up with any other ideas. Ideally, we should provide `--vulndb-dir` (or simply `--vulndb` as `#1` might be supporting Redis as well) and `--bom-cache`, then deprecate `--cache-dir`. Deprecating `--cache-dir` makes sense if many people use Redis, but I'm afraid that it bothers many users who don't use Redis even though there are a few people redis-users.
Any thought?

Reply:
Ah I forgot about vulnerability DB file. All right in this case I think we're good with --cache-backend applicable only to layers cache.